src/patterns/components/footer/base.hbs

```diff
@@ -1,10 +1,10 @@
 ---
 name: Base
-description: |
-  The base footer is intended to be used at the bottom of every page. The base footer includes legal links,
+description:
+  The base Footer is intended to be used at the bottom of every page. The base Footer includes legal links,
   legal icons or small awards, and a paragraph of legal or other type of copy.
 restrictions:
-  - Some screen readers do not announce the footer landmark role so we add
+  - Some screen readers do not announce the Footer landmark role so we add
     role="contentinfo" to the footer HTML element.
   - The awards/icons section is resticted to a maximum of 5 items.
   - The legal link section is resticted to a maximum of 8 links.
```

Comment:
resticted should be restricted on L9 and L10

Reply:
thanks @scottux !!